src/contexts/userContext.tsx

```diff
@@ -9,10 +15,14 @@ import {
 import { API } from 'endpoints'
 import { getRestClient } from 'clients/restClient'
 import { Page, routes } from 'routes'
+import { checkIfAuthorized } from '../utils/session'
 
 export const UserContext = createContext<UserContextValue | null>(null)
 
 export const UserContextProvider = ({ children }: UserContextProviderProps) => {
+  const [authorized, setAuthorized] = useState<boolean>(
+    useMemo(() => checkIfAuthorized(), [])
```

Comment:
Not sure if useMemo is necessary here. useState argument will be used only once on hook init